src/core/radio/trel_interface.cpp

```diff
@@ -407,6 +413,8 @@ void Interface::HandleReceived(uint8_t *aBuffer, uint16_t aLength)
     Get<Link>().ProcessReceivedPacket(mRxPacket);
 
 exit:
+    ++mTrelCounters.mRxPackets;
```

Comment:
maybe not increment the counters if trel is disabled.

Reply:
Same comment as above.